tests/es6_example/submit-comment.js

```diff
@@ -0,0 +1,17 @@
+const expect = require('chai').expect;
+const HomePage = require('../../pages/es6_example/home.page');
+const homePage = new HomePage();
+
+describe('Mocha Spec Sync example', function() {
+
+    it("Submit Comment", function() {
+        homePage.open();
+        let comment = "This is a comment";
```

Comment:
Technically, these `let` statements could all be `const` since they're immutable.  But don't get me started...javascript made a mess and should have stuck with `var` and fixed scoping issues.

Reply:
Yea, about that, do you guys think it would be cleaner and more readable to just do this?:
`homePage.setComment("This is a comment");`
One line of code less and I think the variable doesn't add any extra context as the method name setComment makes it clear what we are doing with that string. And it resolved @sauceaaron 's concern I think...